tests/src/test/scala/tests.scala

```diff
@@ -27,7 +27,7 @@ class tests extends WordSpec with Matchers {
       import org.ensime.pcplod._
       withMrPlod("pcplodtest.scala") { pc =>
         pc.typeAtPoint('result) shouldBe Some("Option[Int]")
-        pc.typeAtPoint('test) shouldBe Some("(n: Int)freestyle.FreeS[F,Int]")
+        pc.typeAtPoint('test) shouldBe Some("(n: Int)cats.free.FreeApplicative[F,Int]")
```

Comment:
looks good. But can you please add an example for the common use case of a `Free` algebra?